pkg/ls/filestring_unix.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2025 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build unix
```

Comment:
is unix a build tag now? I am not caught up I think :-)

Reply:
:)